test_dev.sh

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bash
+
+# Catch any non-flagged use
+if [ $# -eq 0 ]; then
+    echo "Invalid: $(basename "$0") has required flags, use: -h for help"
+    exit 1
+fi
+
+# e: Stops execution in the instance of a command or pipeline error
+# u: Treat unset variables as an error and exit immediately
+set -eu
+
+# Default to FAILURE
+STATUS="FAILURE"
+
+function outcome_and_exit {
```

Comment:
nit: this function doesn't actually exit, it just gets run while the whole bash process is on its way to exiting anyway.

Reply:
Absolutely. This is an artifact from when the function used to also exit, I forgot to re-assess the naming.